lib/connectors/memory.js

```diff
@@ -567,9 +567,9 @@ function applyFilter(filter) {
         return compare(example.neq, value) !== 0;
       }
 
-      if ('between' in example ) {
-        return ( testInEquality({gte:example.between[0]}, value) &&
-                 testInEquality({lte:example.between[1]}, value) );
+      if ('between' in example) {
+        return (testInEquality({ gte:example.between[0] }, value) &&
+                 testInEquality({ lte:example.between[1] }, value));
```

Comment:
Good catch! Apparently eslint doesn't care whether its 2 spaces, 8 spaces or 9 spaces in this case.

Your example looks exactly as what I would write myself, @Amir-61:
>```js
return (testInEquality({ gte:example.between[0] }, value) &&
  testInEquality({ lte:example.between[1] }, value) &&
  testInEquality({ lte:example.between[2] }, value));
```